docs/sources/dashboards/create-reports/_index.md

```diff
@@ -112,24 +114,76 @@ The report creation process is multi-step, but you don't need to complete these
 
 You can also save the report as a draft at any step in the process:
```

Comment:
we're still debating if we want this behavior or only let the user save a report as a draft if it wasn't created yet